migrate.go

```diff
@@ -395,6 +396,50 @@ func (m *Migrate) Version() (version uint, dirty bool, err error) {
 	return suint(v), d, nil
 }
 
+// Check if exists a pending migration
+func (m *Migrate) Check() error {
+	if err := m.lock(); err != nil {
+		return err
+	}
+
+	curVersion, dirty, err := m.databaseDrv.Version()
+	if err != nil {
+		return m.unlockErr(err)
+	}
+
+	if dirty {
```

Comment:
What's your definition of a "pending migration"? The dirty flag is set before a migration is attempted so any running or failed migration will have a dirty flag. I believe this code will find unapplied migrations.